.npmignore

```diff
@@ -10,5 +10,3 @@ node_modules
 cypress.json
 index.html
 webpack.config.js
-dist/ace-diff-light.min.js
```

Comment:
These `.min.js` files are just artifacts from the webpack pipeline and shouldn't be published.

Reply:
You can see them being introduced here https://github.com/ace-diff/ace-diff/blob/master/webpack.config.js#L12-L16

Reply:
you are right, thank you